test/integration/pricing/system-info.tap.js

```diff
@@ -26,7 +26,7 @@ test('pricing system-info aws', function (t) {
   awsRedirect.put('/latest/api/token').reply(200, 'awsToken')
   // eslint-disable-next-line guard-for-in
   for (const awsPath in awsResponses) {
-    awsRedirect.get('/2016-09-02/' + awsPath).reply(200, awsResponses[awsPath])
+    awsRedirect.get(`/latest/${awsPath}`).reply(200, awsResponses[awsPath])
```

Comment:
I appreciate the string interpolation instead of concatenation. Thanks for that small style fix.